README.md

```diff
@@ -24,68 +24,58 @@
   </p>
 </div>
 
-
 ![preview](assets/preview.gif)
 
-## Table of Contents</summary>
-<ol>
-  <li>
-    <a href="#installation">Installation</a>
-    <ul>
-      <li><a href="#one-click-gui">One-click GUI</a></li>
-      <li><a href="#developer">Developer install</a></li>
-    </ul>
-  </li>
-  <li>
-    <a href="#getting-started">Getting started</a>
-  </li>
-</ol>
+## Manuscript
+**Please cite:**
+> **AlphaDIA enables End-to-End Transfer Learning for Feature-Free Proteomics**<br>
+> Georg Wallmann, Patricia Skowronek, Vincenth Brennsteiner, Mikhail Lebedev, Marvin Thielert, Sophia Steigerwald, Mohamed Kotb, Tim Heymann, Xie-Xuan Zhou, Magnus Schwörer, Maximilian T. Strauss, Constantin Ammar, Sander Willems, Wen-Feng Zeng, Matthias Mann<br>
+> bioarxiv.com/XXX.XXX.XXX
 
+alphaDIA of the alphaX ecosystem:
+- alphaPept
+- alphaTims
+- alphaPeptDeep
+- alphaRaw
+- alphaBase
+- directLFQ
 
 ## Installation
 
 AlphaDIA can be installed on Windows, macOS and Linux. Please choose the preferred installation:
 
 * [**One-click GUI installation:**](#one-click-gui-installation) Choose this installation if you only want the GUI and/or keep things as simple as possible.
 
-* [**Library and CLI installation:**](#library-and-cli-installation) Choose this installation if you want to use alphaDIA as a library or its command-line interface. You need to be familiar with CLI tools, Conda and Python.
-It allows access to all available features of alphaDIA.
+* [**Pip installation**](#pip-installation)This version allows you to use alphaDIA as a package within your conda environment. You will only have access to the search engine backend and the command line but not the GUI.
 
 * [**Developer installation:**](docs/installation.md#developer-installation) This installation allows to modify alphaDIA's source code directly. Generally, the developer version of alphaDIA outperforms the precompiled versions which makes this the installation of choice for high-throughput experiments.
 
 * [**Docker installation:**](docs/installation.md#use-the-dockerized-version) Choose this for running alphaDIA in a Docker container, which is useful if you want to run it in a cloud environment.
 
+* [**Slurm installation:**](docs/installation.md#use-the-dockerized-version) Choose this for running alphaDIA in a Docker container, which is useful if you want to run it in a cloud environment.
```

Comment:
As much as I like to be quoted, still I think this text is not what you intended to say ;-)